stl/inc/format

```diff
@@ -2745,12 +2748,42 @@ struct _Arg_formatter {
     }
 };
 
+template <class _Ty>
+constexpr bool _Is_known_size_string = false;

```

Comment:
We conventionally use `inline` for all variable templates in product code (it's simpler to be consistent, instead of asking whether we ever odr-use them). Occurs below.